### PR TITLE
Extended kabisa theming

### PR DESCRIPTION
**Pull request by alianza**
Added Kabisa (Poppins) font
Altered some styles according to new font
Added product view slider kabisa background colors

### CODE REVIEW

**Comment by alianza** [comment]
⚡️ [Lighthouse report](https://storage.googleapis.com/lighthouse-infrastructure.appspot.com/reports/1626090963695-66865.report.html) for the changes in this PR:
| Category | Score |
| --- | --- |
| 🟠 Performance | 59 |
| 🟢 Accessibility | 100 |
| 🟢 Best practices | 100 |
| 🟢 SEO | 92 |
| 🟢 PWA | 92 |
 
*Lighthouse ran on [https://headless-commerce-poc-bkowt74kh-alianza.vercel.app/](https://headless-commerce-poc-bkowt74kh-alianza.vercel.app/)*

**Comment by pascalw** [comment]
It doesn't seem like the Poppins font is actually used @alianza? 😅 

Screenshot 2021-07-14 at 16 26 40

**Comment by alianza** [comment]
> It doesn't seem like the Poppins font is actually used @alianza? 😅
> 
> Screenshot 2021-07-14 at 16 26 40

It does seem like Poppins is used for me. And I can see it change when I disable the styles for it and it goes back to the old front. Are you using FireFox? Maybe it is something browser related?

**Comment by pascalw** [comment]
Are you sure it's actually using Poppins for you? For me it's not actually using Poppins in Brave either.
The CSS declares the Poppins font it seems, but the font is not actually loaded which is why it's falling back to Helvatica in my case.

Screenshot 2021-07-15 at 08 42 10

**Comment by alianza** [comment]
> Are you sure it's actually using Poppins for you? For me it's not actually using Poppins in Brave either.
> The CSS declares the Poppins font it seems, but the font is not actually loaded which is why it's falling back to Helvatica in my case.
> 
> Screenshot 2021-07-15 at 08 42 10

Fixed this! I wasn't actually importing the font which is why it wasn't working. The fallback was different that the default font which is how I got confused. It looks very nice with the new font!

**Comment by pascalw** [comment]
:+1: One thing I didn't notice earlier is that Poppins is only used for body text in the Kabisa theme (see https://kabisa.nl). `Montserrat ExtraBold` should be used for headers. Can you change that still? 😅

**Comment by alianza** [comment]
> 👍 One thing I didn't notice earlier is that Poppins is only used for body text in the Kabisa theme (see https://kabisa.nl). `Montserrat ExtraBold` should be used for headers. Can you change that still? 😅

Done!

File: assets/base.css [hunk 1]
```diff
@@ -1,3 +1,6 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@300&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@800&display=swap');
```
Comment (pascalw) [inline assets/base.css:2]: This is fine for now, but not ideal for performance. See https://csswizardry.com/2020/05/the-fastest-google-fonts/ for background.

Comment (pascalw) [reply assets/base.css:2]: Curious to see the impact on the Lighthouse score.

**Review by pascalw — APPROVED** [review]

File: assets/base.css (hunk 1 above)
Comment (alianza) [reply assets/base.css:2]: Thank for the background info. Will leave this for now. On my own machine I did not perceive a large difference in performance score.

Comment (pascalw) [reply assets/base.css:2]: Fonts can have a pretty big impact on perceived performance, certainly on cellular connections. For this demo this is not a problem though :-)

**Pull request merged**